R/tools.R

```diff
@@ -8,10 +8,10 @@
 #'
 #' `r .docs_list_tools()`
 #'
-#' @param tools Optional names of tools or tool groups to include when
-#'   registering tools. By default all btw tools are included. For example, use
-#'   `tools = "docs"` to include only the documentation related tools, or
-#'   `tools = c("env", "docs", "session")`, etc.
+#' @param ... Optional names of tools or tool groups to include when registering
+#'   tools. By default all btw tools are included. For example, use `tools =
+#'   "docs"` to include only the documentation related tools, or `tools =
+#'   c("env", "docs", "session")`, etc.
```

Comment:
This caught my eye. I guess, seeing the conversion to dots, I would anticipate that the preferred entry point would now be `btw_tools("env", "docs", "session")`, i.e. not named as `tools = ` and not concatenated?

Reply:
Thanks, I think I just missed this during the docs update